src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx

```diff
@@ -48,142 +48,139 @@ AQM does not encompass anomaly detection or AIOps, which are designed to detect
   >
-    Accumulated incident duration is the total sum of minutes that all the incidents accumulated over a period of time. Typically you should compare the current and previous weeks.
+Accumulated incident duration is the total sum of minutes that all the incidents accumulated over a period of time. Typically you should compare the current and previous weeks.
 
-    **Goal:** Reduce the total accumulated minutes of incidents.
-    **Best practices:**
+**Goal:** Reduce the total accumulated minutes of incidents.
 
-    * Do not manually close incidents. Manual closure will skew the real duration of incident length.
-    * Eliminate alerts that do not result in any remediation actions from the recipients.
-    * Improve percent investigated and mean-time-to-investigate KPIs by communicating their importance in improving detection and response times.
-    * Report AQM KPIs to all Stakeholders.
+**Best practices:**
+* Do not manually close incidents. Manual closure will skew the real duration of incident length.
+* Eliminate alerts that do not result in any remediation actions from the recipients.
+* Improve percent investigated and mean-time-to-investigate KPIs by communicating their importance in improving detection and response times.
+* Report AQM KPIs to all stakeholders.
   </Collapser>
 
   <Collapser
     id="kpi-mttc"
     title="Mean time to close (MTTC) KPI"
   >
-    Average duration of incidents within the period of time measured.
+Average duration of incidents within the period of time measured.
 
-    **Goal:** Reduce MTTC
-    **Best practices:**
+**Goal:** Reduce MTTC
 
-    * Do not manually close incidents. Manual closure will skew the real duration of incident length.
-    * Improve Reliability Engineering skills.
-    * Report AQM KPIs to all stakeholders.
+**Best practices:**
+* Do not manually close incidents. Manual closure will skew the real duration of incident length.
+* Improve reliability engineering skills.
+* Report AQM KPIs to all stakeholders.
   </Collapser>
 
   <Collapser
     id="kpi-pct-under-five"
     title="Percent under 5 minutes KPI"
   >
-    Percentage of incidents where the duration of the incident is under five minutes. This can be an indicator of incident flapping.
+Percentage of incidents where the duration of the incident is under five minutes. This can be an indicator of incident flapping.
 
-    **Goal:** Minimize percentage of incidents with short durations
-    **Best practices:**
+**Goal:** Minimize percentage of incidents with short durations.
 
-    * Ensure that conditions are detecting legitimate deviations from expected behavior.
-    * See Baselining and Service Level Management.
-    * Ensure that conditions are detecting legitimate deviations that correlate to business impact or impending business impact.
+**Best practices:**
+* Ensure that conditions are detecting legitimate deviations from expected behavior.
+* See baselining and Service Level Management.
```

Comment:
@AlecIsaacson I'm not sure what the 'baselining' here is referring to. Do you know?

Reply:
No.  Pull that reference out.